examples/substra_core/diabetes_example/run_diabetes.py

```diff
@@ -380,7 +401,8 @@
 }
 
 for client_id, key in local_task_1_keys.items():
-    print(f"Status of task {key} on client {client_id}: {clients[client_id].get_task(key).status}")
+    wait_task(client=clients[client_id], key=key)
```

Comment:
do we need to wait after each task?

Reply:
There is a need to wait for the synchronisation with the orchestrator between each task (cf `wait_for_asset_synchronized` in substra-tests that does that).
As I didn't wanted to write some complicated code, I ended up with this small "wait task" function...

Reply:
ok, thanks for the explanation